rflx/pyrflx/typevalue.py

```diff
@@ -131,10 +132,10 @@ def literals(self) -> Mapping[Name, Expr]:
         raise NotImplementedError
 
     @property
-    def size(self) -> int:
+    def size(self) -> Number:
         size_expr = self._type.size.simplified()
         assert isinstance(size_expr, Number)
-        return size_expr.value
+        return Number(size_expr.value)
```

Comment:
This looks unnecessary since `size_expr` is already a `Number`.

Reply:
Fixed by 7f48161